File: 백준/Bronze/30802. 웰컴 키트/웰컴 키트.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.StringTokenizer;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int n = Integer.parseInt(br.readLine());
        StringTokenizer st = new StringTokenizer(br.readLine());
        int[] size = new int[6];
        for (int i = 0; i < 6; i++) {
            size[i] = Integer.parseInt(st.nextToken());
        }
        StringTokenizer st2 = new StringTokenizer(br.readLine());
        int t = Integer.parseInt(st2.nextToken());
        int p = Integer.parseInt(st2.nextToken());

        int cnt = 0;
        for (int i = 0; i < 6; i++) {
            cnt += (size[i] / t);
            if (size[i] % t != 0) {
                cnt++;
            }
        }
        System.out.println(cnt);
        System.out.println(n / p + " " + n % p);
    }
}

File: 백준/Bronze/30802. 웰컴 키트/웰컴 키트.kt
import java.io.BufferedReader
import java.io.InputStreamReader
import java.util.StringTokenizer

fun main() {
    val br = BufferedReader(InputStreamReader(System.`in`))

    val n = br.readLine().toInt()

    var st = StringTokenizer(br.readLine())
    val s = st.nextToken().toInt()
    val m = st.nextToken().toInt()
    val l = st.nextToken().toInt()
    val xl = st.nextToken().toInt()
    val xxl = st.nextToken().toInt()
    val xxxl = st.nextToken().toInt()

    st = StringTokenizer(br.readLine())
    val t = st.nextToken().toDouble()
    val p = st.nextToken().toInt()

    var tShirt = 0
    tShirt += Math.ceil(s / t).toInt()
    tShirt += Math.ceil(m / t).toInt()
    tShirt += Math.ceil(l / t).toInt()
    tShirt += Math.ceil(xl / t).toInt()
    tShirt += Math.ceil(xxl / t).toInt()
    tShirt += Math.ceil(xxxl / t).toInt()

    println(tShirt)
    println("${n / p} ${n % p}")
}